redfish/main.py

```diff
@@ -124,10 +124,9 @@
 standard_library.install_aliases()
 from builtins import object
 
-
-
 import json
-from urllib.parse import urlparse, urljoin
+import socket
```

Comment:
No, urllib is part of core modules.
By the way I try to sort the modules in the following way:
1- buildin
2- Common external module (request, tortilla)
3- Our modules